packages/desktop-client/src/components/mobile/MobileNavTabs.tsx

```diff
@@ -265,12 +267,18 @@ function NavTab({ Icon: TabIcon, name, path, style, onClick }: NavTabProps) {
         flexDirection: 'column',
         textDecoration: 'none',
         textAlign: 'center',
+        textWrap: 'balance',
         userSelect: 'none',
+        justifyContent: 'center',
         ...style,
       })}
       onClick={onClick}
     >
-      <TabIcon width={22} height={22} />
+      <TabIcon
+        width={22}
+        height={22}
+        style={{ margin: '0 0 5px 0', minHeight: '22px' }}
```

Comment:
Is the 5px margin required here? It's not a huge difference but I preferred the more compact look from before.

Reply:
It's not required, I only felt the text otherwise was too close to the icon. There is no strong opinion on my side so I'll revert it. This may also help with less VRT changes 😄